src/main/java/frc/robot/classes/Kinematics.java

```diff
@@ -81,14 +81,8 @@ public void setPose(Position2D pose) {
 
   private double limitRadians(double radians) {
     double retval = radians;
-
-    while (retval > Math.PI) {
-      retval -= 2 * Math.PI;
-    }
-
-    while (retval < -Math.PI) {
-      retval += 2 * Math.PI;
-    }
+    
+    retval = retval % Math.PI
```

Comment:
This is not equivalent to the previous code and will result in an error if the value given is between 1 PI and 2PI.
If you want to simplify this you will need to do `retval = retval % (2*Math.PI);`